sklearn/tests/test_grid_search.py

```diff
@@ -217,23 +217,6 @@ def test_grid_search_score_method():
                                               scoring='roc_auc').fit(X, y)
     search_auc = GridSearchCV(clf, grid, scoring='roc_auc').fit(X, y)
 
-    # Check warning only occurs in situation where behavior changed:
-    # estimator requires score method to compete with scoring parameter
-    score_no_scoring = assert_no_warnings(search_no_scoring.score, X, y)
-    score_accuracy = assert_warns(ChangedBehaviorWarning,
-                                  search_accuracy.score, X, y)
-    score_no_score_auc = assert_no_warnings(search_no_score_method_auc.score,
-                                            X, y)
-    score_auc = assert_warns(ChangedBehaviorWarning,
-                             search_auc.score, X, y)
-    # ensure the test is sane
```

Comment:
I thought his comment was wanting me to remove it. But if I leave it as it is, the tests fail giving this Failure message : 

```
======================================================================
FAIL: sklearn.tests.test_grid_search.test_grid_search_score_method
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/Ramana/projects/macvnev/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/Ramana/projects/scikit-learn/sklearn/tests/test_grid_search.py", line 224, in test_grid_search_score_method
    search_accuracy.score, X, y)
  File "/Users/Ramana/projects/scikit-learn/sklearn/utils/testing.py", line 135, in assert_warns
    % func.__name__)
AssertionError: No warning raised when calling score


----------------------------------------------------------------------
Ran 34 tests in 1.925s

FAILED (failures=1)
```